sockets/header.js

```diff
@@ -90,35 +88,34 @@ module.exports = function (app, connectionHandler, socket) {
 	this.onInit = function () {
 		// Prevents data wipe
 		this.onConnect();
-
-		async.parallel([
-			getBlockStatus,
-			getPriceTicker,
-		],
-		(err, res) => {
-			if (err) {
-				log('error', `Error retrieving: ${err}`);
-			} else {
-				data.status = res[0];
-				data.ticker = res[1];
-
-				log('info', 'Emitting new data');
-				socket.emit('data', data);
-
-				newInterval(0, 10000, emitData);
-			}
-		});
 	};
 
 	this.onConnect = function () {
-		log('info', 'Emitting existing data');
+		log('debug', 'Emitting existing data');
 		socket.emit('data', data);
 	};
 
 	this.onDisconnect = function () {
-		for (let i = 0; i < intervals.length; i++) {
-			clearInterval(intervals[i]);
-		}
-		intervals = [];
+		log('debug', 'Client disconnected');
 	};
+
+	async.parallel([
+		getBlockStatus,
+		getPriceTicker,
```

Comment:
Same here - `getPriceTicker` should be independent of `blocks/change` event

Reply:
Done. Implemented additional interval-based updates when the block time is exceeded which keeps price ticker up-to-date when forging is stuck.